docs/content/doc/upgrade/from-gitea.en-us.md

```diff
@@ -0,0 +1,85 @@
+---
+date: "2021-09-02T16:00:00+08:00"
+title: "Upgrade from an old Gitea"
+slug: "upgrade-from-gitea"
+weight: 10
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "upgrade"
+    name: "From Gitea"
+    weight: 10
+    identifier: "upgrade-from-gitea"
+---
+
+# Upgrade from an old Gitea
+
+**Table of Contents**
+
+{{< toc >}}
+
+Gitea provides automatically upgrade mechanism. Just get a new Gitea, stop the old one, run the new one.
+Everytime a Gitea instance runs, it checks whether an upgrade action should be taken. 
+If an upgrade action is required, Gitea will take some time to complete the upgrade and then serve.
+
+## Downgrade and backup
+
+Gitea keeps compatibility for versions whose first two fields are the same (`a.b.x` -> `a.b.y`), 
+these versions can be upgraded and downgraded with the same database structure. 
+Otherwise (`a.b.?` -> `a.c.?`), a new Gitea will upgrade an old database 
+to a new structure which may not be used by an old Gitea.
+
+For example:
+
+| From | To |  |
+| --- | --- | --- |
+| 1.4.0 | 1.4.1 | ✅ |
+| 1.4.1 | 1.4.0 | ✅ |
+| 1.4.1 | 1.5.0 | ✅ Database gets upgraded |
+| 1.5.0 | 1.4.1 | ❌ Database already got upgraded and can not be used for an old Gitea |
+
+**Since you can not run an old Gitea with an upgraded database, 
+if stability is the top priority and you want to make sure there is a chance to downgrade,
+a backup should always be made before upgrade.** 
+
+Backup steps:
+
+* Stop Gitea instance
+* Backup database (important)
+* Backup Gitea config (optional)
+* Backup Gitea data files in `APP_DATA_PATH` (optional)
```

Comment:
This is not optional, avatars, attachments and lfs should be stored on the directory defaultly.

Reply:
OK, I will remove these `important` and `optional`, recommend users to do a full backup.